dm/master/status.go

```diff
@@ -39,6 +40,12 @@ func (h *statusHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 func InitStatus(lis net.Listener) {
 	mux := http.NewServeMux()
 	mux.Handle("/status", &statusHandler{})
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
```

Comment:
wo~~ the pprof was disabled because we introduced `mux` instead of the original builtin _http_, I think you can remove `_ "net/http/pprof"` in 3 _main.go_ now.